com.chartboost.helium/Android~/helium-android-bridge/src/main/java/com/chartboost/heliumsdk/unity/HeliumUnityBridge.kt

```diff
@@ -70,6 +69,13 @@ class HeliumUnityBridge {
             }
         }
 
+        initResultsObserver = object : PartnerInitializationResultsObserver {
+            override fun onPartnerInitializationResultsReady(data : PartnerInitializationResultsData) {
+                val json = data.data.toString()
```

Comment:
🤔 Wondering if you really do need to `toString()`.

Reply:
Yes, it is needed since we are passing a JSON string not a JSON object to Unity 👍🏻